.github/workflows/nightly.yml

```diff
@@ -43,13 +43,44 @@ jobs:
       container: ${{ matrix.container }}
       conan_profile: ${{ matrix.conan_profile }}
       build_type: ${{ matrix.build_type }}
-      code_coverage: false
```

Comment:
This flag doesn't exist anymore in the used workflow, need to remove it to make it work.